README.md

```diff
@@ -316,6 +318,49 @@ trait HasTranslations
 }
 ```
 
+### Fallback callback
+
+You can setup a fallback callback that is called when a translation key is missing/not found.
+Setting such a callback doesn't affect fallback behaviour and doesn't need fallback to be enabled.
+It just lets you execute some custom code like logging something or contact a remote service for example.
+
+You need to set it up in `config/translatable.php`:
+
+```
+return [
+    'fallback_callback_enabled' => true,
+    'fallback_callback_class' => App\MyHandler::class,
```

Comment:
Instead of having to add a handler in a config class, I would very much prefer a dedicated class that people can use to configure the package. Could look something like:

```php
// this can be put in a service provider.

Translations::fallback(function($model, string $translationKey, string $locale) {

})
```

This way people can configure fallbacks more easily

Reply:
Wow I agree, it looks way better ! Do you have another package in mind with something similar. It would be great if I can look how it is made. Thanx

Reply:
I went for `Translatable::fallback()`

What do you think ?